package.json

```diff
@@ -45,23 +45,21 @@
 		"clean:docs": "rimraf **/_api-extractor-temp docs/api/*/**",
 		"clean:nyc": "rimraf nyc/**",
 		"clean:r11s": "cd server/routerlicious && npm run clean",
-		"format": "npm run prettier:root:fix && pnpm run -r --no-sort --stream format",
+		"format": "npm run prettier:fix",
 		"format:changed": "npm run format:changed:main",
 		"format:changed:main": "pretty-quick --branch main",
 		"format:changed:next": "pretty-quick --branch next",
 		"preinstall": "node scripts/only-pnpm.cjs",
 		"layer-check": "fluid-layer-check --info build-tools/packages/build-tools/data/layerInfo.json",
-		"lint": "npm run syncpack:deps && npm run syncpack:versions && npm run prettier:root && npm run ci:eslint",
-		"lint:fix": "npm run prettier:root:fix && pnpm run -r --no-sort --stream lint:fix",
+		"lint": "npm run syncpack:deps && npm run syncpack:versions && npm run ci:eslint",
+		"lint:fix": "pnpm run -r --no-sort --stream lint:fix",
 		"policy-check": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json",
 		"policy-check-help": "echo Detect (and error on) policy-check violations, like package.json sorting, copyright headers etc. Excludes assert-short-code. Run the check or \"pnpm flub check policy --listHandlers\" for a full list.",
 		"policy-check:asserts": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json --handler assert-short-codes --fix",
 		"policy-check:fix": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json --excludeHandler assert-short-codes --fix",
 		"policy-check:fix-help": "echo Fix policy-check violations excludes assert-short-code/",
-		"prettier": "npm run prettier:root && pnpm run -r --no-sort --stream prettier",
-		"prettier:fix": "npm run prettier:root:fix && pnpm run -r --no-sort --stream prettier:fix",
-		"prettier:root": "prettier --check ./*.* --ignore-path .prettierignore",
-		"prettier:root:fix": "prettier --write ./*.* --ignore-path .prettierignore",
+		"prettier": "prettier --check . --ignore-path ./.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ./.prettierignore",
```

Comment:
This will this start formatting the server release group when run from the root, right? Is that intended? I think it shouldn't...

Reply:
Yes, it's intended. In the ideal case, this should never be a problem because CI enforces that everything is formatted properly already, so 

If we don't do it this way, where should the script to format the whole repo live?

Reply:
I think my honest answer to that is "here but server should live in a separate repo" 😆. Not strongly opposed to moving forward like this, just noting that my spidey sense reacted. One specific concern I have is if server folks update "their copy" of `.prettierignore` but no the root; we expect CI to fail in that case because it will format from the root and cause the validation for unexpected file changes to complain?

Reply:
I created new prettier:repo and format:repo scripts so the old ones still stay scoped to the client release group.